assets/themes/waste/waste.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "waste",
+  "maintainer": "",
+  "version": "2022-01-19",
+  "language": [
+    "en",
+    "nl"
+  ],
+  "title": {
+    "en": "Waste",
+    "nl": "Afval"
+  },
+  "description": {
+    "en": "Map showing waste baskets and recycling facilities.",
+    "nl": "Kaart met afvalbakken en recyclingfaciliteiten."
+  },
+  "icon": "./assets/layers/recycling/recycling-14.svg",
+  "startZoom": 19,
+  "startLat": 53.24865,
+  "startLon": 6.60075,
+  "layers": [
+    {
+      "builtin": "waste_basket",
+      "override": {
+        "minzoom": 12
+      }
+    },
+    "recycling",
+    "waste_disposal"
```

Comment:
Waste disposal is a bit of a special case, but it is worth considering setting a higher minzoom too as they won't be useful for someone glancing over the map.

Reply:
Minzoom now also 18, still thinking about merging with recycling (need some more sleep first 😉)

Reply:
Haha, yeah, good night!